### DISCUSSION TITLE
Are there low predictions in android wear 2?

### DISCUSSION THREAD

**Discussion by dlopez1984** [NightscoutFoundation/xDrip#3015] [Q&A]
There is any possibility of show low predictions in smartwatch, working as collector?
 It is very useful and I think that is one of the best reasons to have Xdrip

If you can clarify me this point, thank you in advance!

**Comment by kskandispersonal** [comment]
Predictions could be added to the smartwatch collector but processing them will have a big impact on battery usage.  Therefore, the smartwatch may not be the best device to process predictions, at leas as standalone.  Are you looking for a standalone solution where phone is not in range?

**Comment by doc77** [comment]
@kskandispersonal  I think it would be a great idea to add predictions to the watch and make it optional in xDrip prefs.

I personally charge my watch twice a day, and would welcome the ability to show predictions.

**Comment by dlopez1984** [comment]
I was using a finow q1 pro as my collector because this watch is really like a mobile phone and I could watch my predictions in realtime. 

Lastly, I have to change my watch for an smartwatch like Ticwatch pro, and after trying with this smartwatch, I could discover the difference between the app in these scenarios.

**Comment by kskandispersonal** [comment]
@dlopez1984 I see.  Yes, the wear watch app companion only supports a subset of the full xDrip+ app features.

@doc77 The watch display itself is big user of the battery on the watch.  Adding the additional predictions will definitely have an impact!  I'll take a look and see about adding it as an option.

**Comment by kskandispersonal** [comment]
@jamorham Shall I add this to the wear code?  I think it is just a matter of adding the app prediction code in BgGraphBuilder.java, as well as adding the Profile.java and some other missing files to the wear tree.  Hwr, if you are working on these files, it may not be an optimal time to integrate them into the wear tree.  What do you think?  I'd also like to add an option to view the steps graphically on the watch which is also in the app/BgGraphBuilder.java.

**Comment by tankrizzo** [comment]
I'd like this functionality as well please.

**Comment by Navid200** [comment]
I am going to convert this to a discussion.
